src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -179,6 +182,14 @@ def enable_autocomplete(self):
         argcomplete.autocomplete(self, validator=lambda c, p: c.lower().startswith(p.lower()),
                                  default_completer=lambda _: ())
 
+    def _set_manual_recommendations(self, **kwargs):
```

Comment:
Using `kwargs` doesn't seem be to absolute necessary.

Reply:
as addressed above.